src/plugins/auto/tests/unit/select_device_test.cpp

```diff
@@ -249,7 +249,7 @@ class SelectDeviceTest : public ::testing::TestWithParam<ConfigParams> {
                    StrEq(METRIC_KEY(OPTIMIZATION_CAPABILITIES)), _)).WillByDefault(RETURN_MOCK_VALUE(gpuCability));
        ON_CALL(*core, GetMetric(StrEq("MYRIAD"),
                    StrEq(METRIC_KEY(OPTIMIZATION_CAPABILITIES)), _)).WillByDefault(RETURN_MOCK_VALUE(myriadCability));
-       ON_CALL(*core, GetMetric(StrEq(CommonTestUtils::DEVICE_KEEMBAY),
+       ON_CALL(*core, GetMetric(StrEq(CommonTestUtils::DEVICE_NPU),
                    StrEq(METRIC_KEY(OPTIMIZATION_CAPABILITIES)), _)).WillByDefault(RETURN_MOCK_VALUE(vpuxCability));
```

Comment:
Please update device name in comments from VPUX to NPU.

Reply:
Done.